### DISCUSSION TITLE
How to clear the editor content?

### DISCUSSION THREAD

**Discussion by anniesexton** [outline/rich-markdown-editor#316] [General]
I'm trying to figure out how to programmatically clear the editor content. Changing defaultValue to an empty string doesn't seem to do it, and I've read that modifying the `value` property isn't recommended either, correct?

The best I've come up with is by setting the `.innerHTML` of the main `.ProseMirror` div. Is this the best way to go about clearing the editor? This feels very un-React, wondering if there's a better way to do this.

Any and all suggestions are much appreciated!

**Comment by tommoor** [comment]
You should be able to change `value` to clear the editor, It's not recommended to update `value` with every keystroke. You could also change a `key` prop to cause the entire component to remount.

**Comment by anniesexton** [comment]
Gotcha! I'll give that a go, thanks!
